lib/resource/Organization.js

```diff
@@ -49,6 +50,32 @@ Organization.prototype.getDefaultGroupStore = function getDefaultGroupStore(/* [
   return this.dataStore.getResource(this.defaultGroupStoreMapping.href, args.options, OrganizationAccountStoreMapping, args.callback);
 };
 
+Organization.prototype.getDefaultAccountStore = function getDefaultAccountStore(/* [options,] callback */) {
+  var args = utils.resolveArgs(arguments, ['options', 'callback'], true);
+  var dataStore = this.dataStore;
+
+  return this.getDefaultAccountStoreMapping({ expand: 'accountStore' }, function (err, accountStoreMapping) {
+    if (err) {
+      return args.callback(err);
+    }
+
+    dataStore.getResource(accountStoreMapping.accountStore.href, args.options, Directory, args.callback);
```

Comment:
The account store can be a group or directory, so this cannot be hard-coded as a directory.

`OrganizationAccountStoreMapping` inherits from `AccountStoreMapping`, which has a `getAccountStore()` method that can resolve the appropriate account store type.  I'd suggest replacing this `data.getResource` call with `organizationAccountStoreMapping.getAccountStore()`